### ISSUE TITLE
[BUG] Select Component UI issues

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#559]
https://modus-web-components.trimble.com/?path=/docs/user-inputs-select--default

- Select dropdown icon on right isn't the correct one
- input options are not correct font
- Focus state doesn't have blue bottom border

The component isn't a native select component which is causing accessibility issues. The dropdown isn't accessible via keyboard Enter for example.

@ryan-henness-trimble 
Could we consider using a native select instead?

**Comment by sarafarhat13** [comment]
Can we have a placeholder text for this control? Thanks!

**Comment by msankaran0712** [comment]
Also, the `active` class is applied to any option that is hovered over.

**Comment by msankaran0712** [comment]
Added `data-table` tag because the bug should be fixed for Table pagination's page size options

**Comment by HarshithaNous** [comment]
@hemantk1325 adding the below scenario with respect to keyboard accessibility

User will hit on tab key to select the first option on Select component "Select Demo1" and then hit on Enter key for the dropdown values to be displayed. Select any of the option by hitting Enter key, once the option is selected, it is expected that the dropdown should auto close. 
However this works fine when user does the action on mouse click, the same should work for action performed through keyboard.

**Comment by msankaran0712** [comment]
@hemantk1325 In this issue please concentrate on only using native select, we have split the bug into multiple issues
